x/gov/depositsvotes.go

```diff
@@ -72,14 +73,33 @@ func StringToVoteOption(str string) (VoteOption, sdk.Error) {
 }
 
 //-----------------------------------------------------------
+// REST
+
+// Rest Deposits
+type DepositRest struct {
```

Comment:
Let's come up with a naming schema for REST output

Reply:
pulling in @cwgoes @rigelrozanski

Reply:
`DepositRest` is fine IMO